packages/babel-preset-expo/.babelrc

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
I can't replicate this behavior but it seems safe to include the file regardless(?)

Reply:
It's fine to include a dummy .babelrc. Reading the [docs](https://babeljs.io/docs/config-files#file-relative-configuration) I think it might be unnecessary, though. They say: 

> Searching will stop once a directory containing a package.json is found, so a relative config only applies within a single package.

Mainly just want the tests not to run extraneous Babel transformations and to be deterministic.